libuavcan/src/transport/uc_can_acceptance_filter_configurator.cpp

```diff
@@ -204,6 +215,22 @@ uint16_t CanAcceptanceFilterConfigurator::getNumFilters() const
     }
 
     return (out == InvalidOut) ? 0 : out;
+	}
+	else
+	{
+		return filters_number_;
+	}
+}
+
+int16_t CanAcceptanceFilterConfigurator::addFilterConfig(const CanFilterConfig& config)
+{
+    CanFilterConfig supl_config = config;
```

Comment:
I guess without this, the compiler was complaining about the reference being const? The proper fix is as follows: `multiset_configs_.emplace<const CanFilterConfig&>(config)`.